camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/AbstractYamlTest.java

```diff
@@ -125,7 +125,7 @@ protected Entity createAndStartApplication(String... multiLineYaml) throws Excep
     }
     
     protected Entity createAndStartApplication(Reader input) throws Exception {
-        return createAndStartApplication(Streams.readFully(input));
+        return createAndStartApplication(Streams.readFullyAndClose(input));
```

Comment:
Overall, I'm fine with this because it's just in tests. But it feels like the responsibility of the caller to close the Reader that they created, rather than this method.